### ISSUE TITLE
Support CertificateFile

### ISSUE THREAD

**Issue by beyondbill** [stefansundin/terraform-provider-ssh#10]
Our bastion host verifies certificate in addition to SSH key.For example, to establish a SSH tunnel to a private Postgres DB server via bastion, we'll run

```
ssh -N -o CertificateFile=~/.ssh/cert.pub -L 5432:pg.foo.com:5432 user@bastion.foo.com
```

I'd love to see a `certificate_file` option in the tunnel data source. Certificate is supported in the underlying Go lib `golang.org/x/crypto/ssh`. I would appreciate an assessment on whether this feature is relatively easy to implement. If so, I can spend time working on it with some general directions from project owner(s).

**Comment by stefansundin** [comment]
Great suggestion. I haven't used CertificateFile before but I'm excited to try it.

**Comment by stefansundin** [comment]
I just added this and it seems to be working fine in my tests: https://github.com/stefansundin/terraform-provider-ssh/commit/588a0a92c8447a9d16c706e1b1115bcc0eab0ddd

Do you want me to compile a binary for you to test with?

I'll probably release a new version this coming week.

**Comment by beyondbill** [comment]
@stefansundin Wow this is amazing! Thanks so much for quickly adding the feature! I should be able to compile a binary by myself if I still remember how to do so three years ago. Will let you know how it works once I test it. Looking forward to the new release!

**Comment by beyondbill** [comment]
Compiled and tested on my Mac. It worked as expected 🎊

**Comment by stefansundin** [comment]
Released in v0.0.4.
